File: src/main/java/com/assets/board/service/PositionService.java
package com.assets.board.service;

import com.assets.board.model.ib.IBPosition;
import org.springframework.web.multipart.MultipartFile;

import java.util.List;

public interface PositionService {

    List<IBPosition> getAllIBPositions();

    void addPositions(MultipartFile ibPositionsFile);
}

File: src/main/java/com/assets/board/service/PositionService.kt
package com.assets.board.service

import com.assets.board.model.ib.IBPosition
import org.springframework.web.multipart.MultipartFile

interface PositionService {
    fun allIBPositionsStatistics(): MutableList<IBPosition?>?

    fun addPositions(ibPositionsFile: MultipartFile): MutableList<IBPosition?>?
}
